crypto/codec/cmt.go

```diff
@@ -39,6 +43,12 @@ func ToCmtProtoPublicKey(pk cryptotypes.PubKey) (cmtprotocrypto.PublicKey, error
 			},
 		}, nil
 	case *secp256k1.PubKey:
+		return cmtprotocrypto.PublicKey{
+			Sum: &cmtprotocrypto.PublicKey_Secp256K1Uncompressed{
+				Secp256K1Uncompressed: pk.Key,
+			},
+		}, nil
+	case *secp256k1.PubKeyOld:
```

Comment:
This case should also convert to `PublicKey_Secp256K1Uncompressed` (check @Raneet10 's PR on peppermint)

Reply:
Thanks, addressed here e3a7a8e305e1fe06f74f57eb7257c246e29130d7

Reply:
Resolved